pkg/reconcile/pipeline/builder/pipeline.go

```diff
@@ -25,10 +27,11 @@ func (i *impl) Process(binding interface{}) (retry bool, err error) {
 	}()
 	ctx, err := i.ctxProvider.Get(binding)
 	if err != nil {
-		return false, err
+		return false, time.Duration(0), err
 	}
 	var status pipeline.FlowStatus
-	for _, h := range i.handlers {
+	for i, h := range i.handlers {
+		fmt.Printf("%v\n", i)
```

Comment:
Forgot to remove it? I guess this is added for debugging.

Reply:
Yep, added this for debugging.  Should be fixed.